doc/project/checklist.md

```diff
@@ -491,7 +491,7 @@ For a transition from S0 to S1, the following items are expected be completed.
 
 ### DIF_USED_IN_TREE
 
-All existing in-tree code which uses the device, uses the device via the DIF.
+All existing non-production in-tree code which uses the device, uses the device via the DIF.
```

Comment:
Maybe a small change to make it clear that other drivers are also acceptable?

`All existing non-production code in the tree which uses the device does so via a DIF or an alternative driver.`